index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hikebuddies - Login</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Mulish:ital,wght@0,200..1000;1,200..1000&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="dist/css/index.css">
+</head>
+<body class="bg-login-body-green-white">
```

Comment:
shouldn't we use semantic tags, like header, main, navbar, footer, etc?

Reply:
there isn't really a navbar and footer on this site, only a logo, so hence that it's not added, but I will add it :) Footer is not in place as of now.